File: sykepengesoknad/src/main/java/no/nav/syfo/kafka/sykepengesoknad/dto/SporsmalDTO.java
package no.nav.syfo.kafka.sykepengesoknad.dto;

import lombok.Builder;
import lombok.Value;

import java.util.List;

@Value
@Builder(toBuilder = true)
public class SporsmalDTO {
    String id;
    String tag;
    String sporsmalstekst;
    String undertekst;
    String min;
    String max;
    SvartypeDTO svartype;
    VisningskriteriumDTO kriteriumForVisningAvUndersporsmal;
    List<SvarDTO> svar;
    List<SporsmalDTO> undersporsmal;
}

File: sykepengesoknad/src/main/kotlin/no/nav/syfo/kafka/sykepengesoknad/dto/SporsmalDTO.kt
package no.nav.syfo.kafka.sykepengesoknad.dto

data class SporsmalDTO(
        val id: String? = null,
        val tag: String? = null,
        val sporsmalstekst: String? = null,
        val undertekst: String? = null,
        val min: String? = null,
        val max: String? = null,
        val svartype: SvartypeDTO? = null,
        val kriteriumForVisningAvUndersporsmal: VisningskriteriumDTO? = null,
        val svar: List<SvarDTO>? = null,
        val undersporsmal: List<SporsmalDTO>? = null
)
